server/errors/ApplicationErrors.js

```diff
@@ -0,0 +1,26 @@
+
+
+class ApplicationError extends Error {
+  constructor(message) {
+    super();
+    Error.captureStackTrace(this, this.constructor);
+    this.name = this.constructor.name;
+    this.message = message || 'something went wrong. Please try again.';
+  }
+}
+
+class AuthenticationError extends ApplicationError {
+  constructor(message) {
+    super(message || 'Incorrect email and password combination.');
+  }
+}
+class ServerError extends ApplicationError {
+  constructor(message) {
+    super(message || 'There was a server error. Please contact the site administrator.');
+  }
+}
+
```

Comment:
I created this so the user would get a more sensible error and not an application error.

Reply:
Do you remember the conclusion of this conversation? Didn't we agree that we don't want to actually send error objects? I feel like this is better handled in a catchall for the routes.

Reply:
Do you remember where we left off with these extensions of the error class? My thinking is that this would be better handled in a "catchall" sort-of route. It's been a while. I'm honestly forgetting where we ended up with this.

Reply:
This file should be deleted. Thanks for catching that!